demo2/app.js

```diff
@@ -10,9 +10,20 @@ app.use(express.json());
 app.use(express.urlencoded({extended:false}));
 app.use(morgan('dev'));
 
+// Database
+const db = require('./db/db');
+
+try {
+    db.sequelize.authenticate();
+    console.log('Connection has been established successfully.');
```

Comment:
authenticate is async operation so it has to be wrapped with .then .catch or async/await
log will be presented in any case of fail/success of DB connect